### PR TITLE
Week 6 Updates

### PR DESCRIPTION
**Pull request by abhirupr123**
DIKSHA

Digital Learners' Passbook

### Changes

Added week 6 updates

### Notes

If there are any further notes.

### CODE REVIEW

File: docs/2023/DIKSHA/Digital Learners Passbook/updates/2023-08-11.md [hunk 1]
```diff
@@ -4,13 +4,19 @@ author: Abhirup Roy
...
+  <source src="/Digital Learners Passbook Updated.mp4"/>
+</video>
+
 ## Contributions
 
+https://github.com/abhirupr123/digital-learners-passbook/commit/b464d0c0570bd318b14d8a8e9500b7e72db68792
+
+https://github.com/abhirupr123/digital-learners-passbook/commit/7e63da5c0b3bc9430d2b30f29d538e658b166e52
+
 ## Learnings
```
Comment (photon0205) [inline docs/2023/DIKSHA/Digital Learners Passbook/updates/2023-08-11.md:22]: Add learnings

File: docs/2023/DIKSHA/Digital Learners Passbook/updates/2023-08-11.md [hunk 2]
```diff
@@ -4,13 +4,19 @@ author: Abhirup Roy
...
 
 ## Screenshots / Videos 
 
+<video controls width="100%">
+  <source src="/Digital Learners Passbook Updated.mp4"/>
+</video>
+
 ## Contributions
 
+https://github.com/abhirupr123/digital-learners-passbook/commit/b464d0c0570bd318b14d8a8e9500b7e72db68792
```
Comment (photon0205) [inline docs/2023/DIKSHA/Digital Learners Passbook/updates/2023-08-11.md:18]: keep all the contributions in points

Comment (abhirupr123) [reply docs/2023/DIKSHA/Digital Learners Passbook/updates/2023-08-11.md]: > keep all the contributions in points

Made all the changes

Comment (abhirupr123) [reply docs/2023/DIKSHA/Digital Learners Passbook/updates/2023-08-11.md]: > > keep all the contributions in points
> 
> Made all the changes

https://github.com/Code4GovTech/c4gt-milestones/pull/600/commits/3c8c0209c04ac60bc5d98290c56599de09722617

**Pull request merged**